github-profiles/kenichi-7356/main.css

```diff
@@ -0,0 +1,51 @@
+@import 'https://fonts.googleapis.com/css?family=roboto:300,400,500';
+.header {
```

Comment:
You aren't using any `header` tag on your `.html` file. It's better don't have unused tags on your `.css` file

Reply:
Thank you! Thank you! The feedback you give me is very useful, I already incorporated the changes